pkg/util/users/users.go

```diff
@@ -152,6 +158,21 @@ func (strategy DefaultUserSyncStrategy) ExecuteSyncRequests(requests []spec.PgSy
 	return nil
 }
 
+func resolveOwnerMembership(dbOwner spec.PgUser, additionalOwners []string, db *sql.DB) error {
+	errors := make([]string, 0)
+	for _, additionalOwner := range additionalOwners {
+		if err := revokeRole(dbOwner.Name, additionalOwner, db); err != nil {
```

Comment:
Does revoking an additional role from the role that lacks it generate an error in Go ?
sample psql session:
```
postgres=# revoke cron_admin from pooler;
WARNING:  role "pooler" is not a member of role "cron_admin"
REVOKE ROLE
```

Reply:
Oh, good point. But actually, even better if Postgres does not return an error if there's no wrong membership in place.